lib/workers/repository/dependency-dashboard.ts

```diff
@@ -286,6 +286,11 @@ export async function ensureDependencyDashboard(
   }
   let issueBody = '';
 
+  if (config.dependencyDashboardHeader?.length) {
```

Comment:
I guess your PR will fix this issue?

- #32104

If you're closing that issue, please add this text to your PR's body text (the top post):

`Closes #32104`

This way GitHub automatically closes the linked issue when your PR gets merged.

Reply:
I added it